model/src/commonMain/kotlin/io/github/droidkaigi/feeder/Lang.kt

```diff
@@ -0,0 +1,10 @@
+package io.github.droidkaigi.feeder
+
+import java.util.Locale
```

Comment:
@yetanothercheer  @mhidaka 
sorry for comment to closed PR 🙇 
in commonMain module should't use `java.util`, right? (because, ios moudle won't be able to use this)

Reply:
👀 @RyuNen344  if you consider sharing a module with iOS, It is need some kind of another ways.
we think this approach is fine for the PR. Thank you for your great opinion 👍 

Can you make a new issue ✍ if you like ?

Reply:
@mhidaka 
thank you for reply! I will create a issue!